components/cam/src/utils/cam_pio_utils.F90

```diff
@@ -1089,13 +1091,18 @@ subroutine cam_pio_createfile(file, fname, mode_in)
     integer                                   :: ierr
     integer                                   :: mode
     
-    mode = ior(PIO_CLOBBER, PIO_64BIT_OFFSET)
+    ! Get mode to open file in. The default is to get this from shr_pio_mod,
+    ! but an optional mode_in argument is allowed to override this.
     if (present(mode_in)) then
-      mode = ior(mode, mode_in)
+      mode = mode_in
+    else
+      mode = shr_pio_getioformat(atm_id)
```

Comment:
You might also want to add PIO_CLOBBER by default here

Reply:
Maybe with something like `ior(mode, PIO_CLOBBER)` after this logical?

Reply:
Yes that should work
```
mode = ior(mode, PIO_CLOBBER)
```